concent_api/concent_api/urls.py

```diff
@@ -1,29 +1,16 @@
-"""concent_api URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/1.11/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.conf.urls import url, include
-    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
-"""
 from django.conf            import settings
 from django.core.exceptions import ImproperlyConfigured
 
 from concent_api.constants  import AVAILABLE_CONCENT_FEATURES
 
 urlpatterns = []
 
+# Check for CONCENT_FEATURES variable existence
 if not hasattr(settings, 'CONCENT_FEATURES'):
     raise ImproperlyConfigured("CONCENT_FEATURES variable must be defined in django settings.")
 
 assert set(settings.CONCENT_FEATURES) - set(AVAILABLE_CONCENT_FEATURES.keys()) == set()
 
+# Run project only with apps specified in CONCENT_FEATURES variable
```

Comment:
Those are not apps. Please don't call them that to avoid confusion with `INSTALLED_APPS`.